waku/v2/waku_discv5.nim

```diff
@@ -185,6 +158,66 @@ proc findRandomPeers*(wd: WakuDiscoveryV5, pred = none(WakuDiscv5Predicate)): Fu
 
   return discoveredRecords
 
+proc runLoop(wd: WakuDiscoveryV5, peerManager: PeerManager, record: Option[enr.Record]) {.async.} =
```

Comment:
Discv5 should not know what a peer manager is but IDK how to handle that.

In Rust I would probably use a channel so that any discovered peers could be sent to the peer manager by any discovery service.